lib/pom.rb

```diff
@@ -73,8 +73,8 @@ def initialize( name, version, default_spec = true )
   #            :id => 'gem-staging' )
 
   plugin( :clean,
-          :filesets => [ { :directory => '${basedir}/ruby/gems/shared/specifications/default',
-                           :includes => [ '*' ] },
+          :filesets => [ { :directory => '${basedir}/ruby/gems',
```

Comment:
it is safe, but people expect their gems to stayed installed even after a clean build. at least some people do so :)

so I would keep it as it currently is